popcoder/video.py

```diff
@@ -25,7 +26,7 @@ def __init__(self, data, out, background_color, size=(1280, 720)):
         Constructor
         @param data : The popcorn editor project json blob
         """
-        self.DELETE_VIDEOS = True
+        self.DELETE_VIDEOS = False
```

Comment:
why not delete?
Also, why is this in uppercase? And why is it not something you set in the `__init__` signature?